dev/tests/integration/testsuite/Magento/Payment/_files/disable_all_active_payment_methods.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+use Magento\Config\Model\Config;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+
+$processConfigData = function (Config $config, array $data) {
+    foreach ($data as $key => $value) {
+        $config->setDataByPath($key, $value);
+        $config->save();
```

Comment:
Looks like we don't need to call save after each iteration

Reply:
> Looks like we don't need to call save after each iteration

@naydav , I've debugged your suggestions and I want inform you that we should use  ```$config->save();``` after each ```$config->setDataByPath($key, $value);``` if we want to save all config params.

By the way, the same approach uses in:
- dev/tests/integration/testsuite/Magento/Braintree/_files/payment_configuration.php
- dev/tests/integration/testsuite/Magento/Paypal/Fixtures/process_config_data.php
- dev/tests/integration/testsuite/Magento/SendFriend/Fixtures/process_config_data.php
- dev/tests/integration/testsuite/Magento/Signifyd/_files/website_configuration.php

Please approve and merge ;)

Thank you!